packages/react-chat/src/contexts/RuntimeContext/useRuntimeState.ts

```diff
@@ -131,6 +130,8 @@ export const useRuntimeState = ({ assistant, config }: Settings) => {
 
   const close = () => {
     broadcast({ type: BroadcastType.CLOSE });
+    saveSession(assistant.persistence, config.verify.projectID, sessionRef.current);
```

Comment:
I don't think this is necessary, we save the session on every single interaction already,
https://github.com/voiceflow/react-chat/blob/a3233a6e188504f10c5cf24cb325f8856e965968/packages/react-chat/src/contexts/RuntimeContext/useRuntimeState.ts#L110

The behavior above is uninterruptable as well so it's guaranteed I suppose.

Reply:
@ninabondar I think without this something was breaking, do you recall?

Reply:
for history:  autostart would break on embedded, since there was nothing to 'close' afair